### ISSUE TITLE
Setup Mailchimp

### ISSUE THREAD

**Issue by halfempty** [everythingtype/kaufman#5]
@gehalber The Contact page needs a newsletter form. Does Dean have one going already? Is it with one of the services (ie. MailChimp)? Or do we need to set one up?

**Comment by gehalber** [comment]
He was going to use Mailchimp, but ran into an issue so now using MadMimi. Do you think that could work?

**Comment by halfempty** [comment]
@gehalber Can he provide us with his log-in info for MadMimi or whichever service he's using?

**Comment by gehalber** [comment]
Yes. Emailing you... :)

**Comment by gehalber** [comment]
Before you put that into affect, Dean may want to set up the mailchimp afterall. I'll let you know when I find out from him.

**Comment by gehalber** [comment]
@halfempty 
Yup, definitely moving to Mailchimp. Sending you an email with credentials.

**Comment by halfempty** [comment]
Okay, this is in.

Note that it opens in a new window to complete. The alternative, processing it on the page, is a lot more work, so I'd tend to leave it this way unless it's a problem.
